docs/src/examples/FOBP_example2.2.jl

```diff
@@ -1,29 +1,26 @@
-# # Foundations of Bilevel Programming: Example 2.1
```

Comment:
What happened here? We don't want to modify this file in this PR. Do it in a different one.